modules/gitdata/blob.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
```

Comment:
This module is important to extract many git operations from `models`. Good! But we have `modules/git` and now `modules/gitdata`, maybe other better name for it?

Reply:
Any suggestions for a better name? I named it `gitdata` due to GitHub's API section for blobs and trees: https://developer.github.com/v3/git/

Reply:
I think there's a few more packages like this. 

Basically we have modules that depend on models and modules that models depend on. I wonder if we need a split in the names.

Perhaps backend and modules?

Then this package could simply be modules/git and the other one backend/git?

Reply:
I just dissolved this and put it in repofiles